### DISCUSSION TITLE
[2.2.1 GUI] Crack folder selection fix

### DISCUSSION THREAD

**Discussion by BigBoiCJ** [BigBoiCJ/SteamAutoCracker#73] [Announcements]
- Fixed a bug related to folder selection that made SAC unable to crack any game
  - Really sorry for not testing this enough, it's completely my fault

SteamAutoCracker GUI requires a 64 bits Windows computer.

<hr /><em>This discussion was created from the release <a href='https://github.com/BigBoiCJ/SteamAutoCracker/releases/tag/2.2.1-gui'>[2.2.1 GUI] Crack folder selection fix</a>.</em>

**Comment by soiberi1** [comment]
This still doesn't work, I was trying to crack the lunar remastered collection app id 3255380 
It said : An error has occured and stuck at 1/2 retrieving game

**Reply by BigBoiCJ** [reply]
Please create an issue for this @soiberi1 

Could you also provide the full error logs?

**Comment by AnimeWaifuGirl** [comment]
do not download this zip because it is a virus! 
https://www.virustotal.com/gui/file/c4949e748348256c219ec0406d2e16e3f4845fe676d3dc26dbfcb2059afdfa81

**Reply by BigBoiCJ** [reply]
You don't know what you're talking about #106 

https://github.com/BigBoiCJ/SteamAutoCracker?tab=readme-ov-file#virus-detection

**Comment by MasterFoann** [comment]
Friend used this and got hacked on multiple platforms. Unless you can prove otherwise, this is a trojan. I have no idea how this has so many stars.

**Reply by BigBoiCJ** [reply]
Got hacked through something else. I don't know, something like downloading pirated games from unsafe websites? Or running .exe files from random people on discord?

The source for all DLL cracks is given in the repo. Steamless is directly from the steamless project. The python code is open source and the compiled .exe is just built with pyinstaller.

The entire environment can easily be replicated with this: https://github.com/BigBoiCJ/SteamAutoCracker/tree/compile-env

This tool is approved and used by a lot of people: the r/PiratedGames megathread and the cs.rin.ru post are proof of it

Questioning the safety of the software is a good practice however, but please don't spread lies. You're free to use an alternative. There are a lot of them. This project, however, might be the easiest one to review as it is simple and doesn't have many dependencies, which might be the reason for its inclusion in r/PiratedGames.

**Comment by Karl5502** [comment]
It doesnt work i dont even know what im doing wrong

**Comment by kanftud** [comment]
"updating app list" Is stuck and keeps retrying infinitely on any possible setting, is there any manual way to get the app list?

<img width="807" height="257" alt="copie" src="https://github.com/user-attachments/assets/7508cdaf-e366-4b46-afe2-d33101873bc3" />

**Reply by Geekshere** [reply]
Just search the app id on steamdb and paste that in instead, this has been an issue for a while and the project is dead
